eth/v1alpha1/beacon_chain.proto

```diff
@@ -556,24 +556,32 @@ message ActiveSetChanges {
 message ValidatorPerformanceRequest {
     // A list of 48 byte validator public keys.
     repeated bytes public_keys = 1;
-
-    // The slot at which the performance is being requested.
-    uint64 slot = 2;
 }
 
 message ValidatorPerformanceResponse {
     // A list of validator balances mapped 1-to-1 with the request's
 	// public keys.
     repeated uint64 balances = 1;
-	// The total number of validators in the beacon chain.
-    uint64 total_validators = 2;
-	// The total number of active validators in the beacon chain.
-    uint64 total_active_validators = 3;
+    // The slot of when validator's attestation got included in the chain at previous epoch, the slot
+    // is mapped 1-to-1 with the request's public keys.
+    repeated uint64 inclusionSlots = 2;
```

Comment:
@terencechain can we use snake case as convention for protos? inclusion_slots

Reply:
Done, sorry about that. Written too much Go lately :P